### DISCUSSION TITLE
mcbbs整合包下载出错，以及正版登录无法使用，但无法知道原因

### DISCUSSION THREAD

**Discussion by UnKn0wnFLAME** [Meloong-Git/PCL#610] [提问] [answered]
[Log2.txt](https://github.com/Hex-Dragon/PCL2/files/10247662/Log2.txt)
[Log3.txt](https://github.com/Hex-Dragon/PCL2/files/10247663/Log3.txt)
[Log4.txt](https://github.com/Hex-Dragon/PCL2/files/10247664/Log4.txt)
[Log5.txt](https://github.com/Hex-Dragon/PCL2/files/10247665/Log5.txt)
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/10247667/Log1.txt)
游戏安装失败
你的网络环境不佳，导致难以连接到服务器。请重试，或尝试使用 VPN。
————————————
详细错误信息：
→ 下载原版 1.18.2失败
→ 下载原版 Json 文件失败
→ 文件下载失败：D:\minecraft\新建文件夹 (2)\.minecraft\versions\Deuterium for Minecraft 1.18.2\Deuterium for Minecraft 1.18.2.json
基础连接已经关闭: 发送时发生错误。（https://piston-meta.mojang.com/v1/packages/23b6e0e0d0f87da36075a4290cd98df1a76e2415/1.18.2.json）
→ 基础连接已经关闭: 发送时发生错误。
→ 无法从传输连接中读取数据: 远程主机强迫关闭了一个现有的连接。。
→ 远程主机强迫关闭了一个现有的连接。
   在 PCL.ModNet.NetFile.Thread(NetThread Info)
错误类型：System.Net.Sockets.SocketException
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
**我下载curseforge的整合包安装没有任何问题，但使用mcbbs的或者是mc百科的整合包就会无法下载json文件，不知道是为什么，而且正版也无法登录**
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
![image](https://user-images.githubusercontent.com/86244709/208152471-0a0baf03-01f9-431d-8406-2ede216a9d2b.png)

**Comment by WForst-Breeze** [comment, marked as answer]
目测就单纯是网络问题，因为你在下载CurseForge整合包的时候日志显示也有网络不佳而无法连接到服务器的时候qwq……如果仍无法成功安装，请尝试使用VPN
